ingestion_server/ingestion_server/authority.py

```diff
@@ -51,7 +47,8 @@ class AuthorityTypes(Enum):
     'museumsvictoria': AuthorityTypes.CURATED,
     'met': AuthorityTypes.CURATED,
     'mccordsmuseum': AuthorityTypes.CURATED,
-    'digitaltmuseum': AuthorityTypes.CURATED
+    'digitaltmuseum': AuthorityTypes.CURATED,
+    'clevelandmuseum': AuthorityTypes.CURATED
```

Comment:
there are a few other museums not given an authority type, is there a particular reason why? thinking `thorvaldsensmuseum` and `brooklynmuseum` in particular

Reply:
That's an error. I'll add them before merging